modules/react/tabs/stories/Tabs.stories.mdx

```diff
@@ -108,84 +106,151 @@ the end.
 
 ### Tabs
 
-The `Tabs` component is the container component that is responsible for creating a `TabsModel` that
-is shared using React context. It does not represent a real element.
-
 #### Usage
 
+`Tabs` is a container component that is responsible for creating a `TabsModel` and sharing it with
+its children using React context. It does not represent a real element.
+
 ```tsx
 // configuration
-<Tabs onActivate={({ data }) => console.log('Activate tab', data.tab)}>
-  {/* sub components go here */ }
-</Tabs>
+<Tabs onActivate={({data}) => console.log('Activate tab', data.tab)}>
+  {/* subcomponents go here */}
+</Tabs>;
 
 // OR
 
-// model
-const model = useTabsModel(config)
+// hoisted model
+const model = useTabsModel(config);
 
-<Tabs model={model}>
-  {/* sub components go here */ }
-</Tabs>
+<Tabs model={model}>{/* subcomponents go here */}</Tabs>;
 ```
 
-#### Component Props
+#### Props
 
 <PropsTable of={Tabs} />
 
----
-
 ### Tabs.List
 
-`Tabs.List` is a `div` element with a `role=tablist`. It is a container for `Tabs.Item` components.
-Any additional attributes provided will be forwared to the dom element.
-
 #### Usage
 
+`Tabs.List` is a `div` element with a `role=tablist`. It is a container for `Tabs.Item`
+subcomponents.
+
 ```tsx
 <Tabs.List>{/* Tabs.Items go here */}</Tabs.List>
 ```
 
-#### Component Props
+#### Props
 
-<PropsTable of={Tabs.List} />
+Undocumented props are spread to the underlying `div` element.
 
----
+<PropsTable of={Tabs.List} />
 
 ### Tabs.Item
 
+#### Usage
+
 `Tabs.Item` is a `button` element with a `role=tab`. Each `Tabs.Item` is associated with a
 `Tabs.Panel` either by a `name` attribute or by the position in the list. If a `name` is provided,
-it must match the name passed to the `Tabs.Panel` component.
-
-#### Usage
+it must match the name passed to the corresponding `Tabs.Panel` component.
 
 ```tsx
 <Tabs.Item name="first">First Tab</Tabs.Item>
 ```
 
-#### Component Props
+#### Props
 
-<PropsTable of={Tabs.Item} />
+Undocumented props are spread to the underlying `button` element.
 
----
+<PropsTable of={Tabs.Item} />
```

Comment:
I noticed the use of both `PropsTable` and `ArgsTable`. I think we need to pick one.